scitasexternal/packages/ansys/package.py

```diff
@@ -69,10 +69,12 @@ def setup_environment(self, spack_env, run_env):
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'autodyn/bin'))
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'fluent/bin'))
         run_env.prepend_path('PATH', join_path(ansys_prefix, 'polyflow/bin'))
-        run_env.prepend_path('PATH', join_path(ansys_prefix, 'tgrid/bin'))
+        if '17.1' in ansys_prefix: run_env.prepend_path('PATH', join_path(ansys_prefix, 'tgrid/bin'))
```

Comment:
I would use a spack builtin mechanism to check the version here.
```suggestion
        if version == Version('17.1'): run_env.prepend_path('PATH', join_path(ansys_prefix, 'tgrid/bin'))
```